modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -72,7 +80,7 @@ EOM
 ewogInVybCI6ICJodHRwczovL3RhbmcuZXhhbXBsZS5jb20iLAogInRocCI6ICJaUk1leTFjR3cwN3psVExHYlhuUWFoUzBHdTAiCn0K
 ----
 
-. Replace the “source” in the TPM2 example with the Base64 encoded file for the type of node to update:
+. Replace the “source” in the TPM2 example with the Base64 encoded file for the type of node to update.
```

Comment:
It is probably beyond the scope of this PR, but I think this step is a bit confusing if you haven't read the TPM2 procedure. Ideally I think this Tang procedure should not refer to the files in the TMP2 procedure. Instead this should say something like 'In the `openshift` directory, create master or worker files to encrypt disks for those nodes' similar to the example in step 3 in that other procedure.

I also think that the `./99-openshift-master-encryption.yaml` file should be called `./99-openshift-master-tang-encryption.yaml` to follow the same format as the other files. But again, that might be out of scope in this PR.

Reply:
Good point. It was out of scope, yes, though I do remember having a similar thought about "TPM2" when first reading through this. I've gone ahead and made the updates as you have illustrated. Very helpful!

Reply:
This is great, thanks Bob.